index.js

```diff
@@ -0,0 +1,13 @@
+const http = require('http');
+require('dotenv').config();
+
+const PORT = 3000;
```

Comment:
this variable is not using in your code (you use `process.env.PORT`), so lets remove it